KsApi/models/PushEnvelope.swift

```diff
@@ -34,6 +35,13 @@ public struct PushEnvelope {
     public let projectId: Int
   }
 
+  public struct PledgeRedemption {
+    public let id: Int
+    public let backingId: Int
```

Comment:
I'm actually not sure that we need to decode backingId. Adding it for now while I wait for confirmation.

Reply:
Yeah, I don't think we need the backingId for the webview - my guess is this is included because we would've needed it for a native implementation.